src/test/java/com/endava/TestCreativeAndUXPage.java

```diff
@@ -15,53 +15,167 @@
+        creativeAndUXPage.assertUrlEndsWith("Creative-And-UX");
+        creativeAndUXPage.clickOnTypeOfMessage();
+        creativeAndUXPage.enterEmail("indira.gandhi@delhi.com");
+        creativeAndUXPage.enterCountry("India");
+        creativeAndUXPage.validateEmail("indira.gandhi@delhi.com");
+        creativeAndUXPage.validateCountry("India");
+        creativeAndUXPage.enterFirstName("Indira");
+        creativeAndUXPage.validateFirstName("Indira");
+        creativeAndUXPage.clickOnCotactUsButton();
+        creativeAndUXPage.assertLastNameWarningMessageIsShown();
+
+        logger.info("Test testCreativeAndUXPage end");
+    }
 
 	/**
 	 * Go to endava.com. Scroll down to the page footer and from the "Services"
 	 * section click on "Creative & UX". Validate URL has changed. Scroll down to
-	 * the "Contact us" area. Populate Email Address field. Populate Country/Region
-	 * Name field. Validate Email Address and Country/Region name are correct. Click
+	 * the "Contact us" area. Populate Last name, Country/Region and Email.
+	 * Validate last name, Email Address and Country/Region name are correct. Click
 	 * on "Contact US" button. Validate that for all mandatory fields there is a
 	 * warning message if it's not populated (e.g. "Please enter First Name").
 	 */
-	@Test
-	public void testCreativeAndUXPage() {
-		logger.info("Test testCreativeAndUXPage start");
-
-		creativeAndUXPage = homePage.openCreativeAndUXPage();
-		creativeAndUXPage.assertUrlEndsWith("Creative-And-UX");
-		creativeAndUXPage.enterEmail("indira.gandhi@delhi.com");
-		creativeAndUXPage.enterCountry("India");
-		creativeAndUXPage.validateEmail("indira.gandhi@delhi.com");
-		creativeAndUXPage.validateCountry("India");
-		creativeAndUXPage.clickOnCotactUsButton();
-		creativeAndUXPage.assertFirstNameWarningMessageIsShown();
-		creativeAndUXPage.assertLastNameWarningMessageIsShown();
-		creativeAndUXPage.assertCompanyMissingWarningMessageIsShown();
-		creativeAndUXPage.assertTermsAndConditionsWarningMessageIsShown();
-
-		logger.info("Test testCreativeAndUXPage end");
-	}
-
-	@AfterMethod
-	public void tearDown() {
-		homePage.quit();
-	}
+    @Test
+    public void testCreativeAndUXPage3() {
+        logger.info("Test testCreativeAndUXPage start");
+
+        creativeAndUXPage = homePage.openCreativeAndUXPage();
+        creativeAndUXPage.assertUrlEndsWith("Creative-And-UX");
+        creativeAndUXPage.clickOnTypeOfMessage();
+        creativeAndUXPage.enterEmail("indira.gandhi@delhi.com");
+        creativeAndUXPage.enterCountry("India");
+        creativeAndUXPage.validateEmail("indira.gandhi@delhi.com");
+        creativeAndUXPage.validateCountry("India");
+        creativeAndUXPage.enterLastName("Gandhi");
+        creativeAndUXPage.validateLastName("Gandhi");
+        creativeAndUXPage.clickOnCotactUsButton();
+        creativeAndUXPage.assertFirstNameWarningMessageIsShown();
+        //creativeAndUXPage.assertCompanyMissingWarningMessageIsShown();
```

Comment:
Why is this commented out? Please do not leave commented out lines in the code.

Reply:
Done.